sdk/servicebus/service-bus/samples/javascript/advanced/sessionRoundRobin.js

```diff
@@ -83,7 +83,7 @@ async function receiveFromNextSession(serviceBusClient) {
   } catch (err) {
     if (
       isServiceBusError(err) &&
-      (err.reason === "SessionCannotBeLocked" || err.reason === "OperationTimeout")
+      (err.reason === "SessionCannotBeLocked" || err.reason === "ServiceTimeout")
```

Comment:
There is no "OperationTimeout" in the literals for `ServiceBusError.reason` and was leading to a compile error since this path would be unreachable.

Hence updating to "ServiceTimeout".
/cc @richardpark-msft let me know if you suggest something else.

Reply:
Good fix, we can talk about how they're classified. I took a shot at it and they might not be right.